spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Foods Index (search results)' do
+  describe 'happy path' do
+    it 'user sees search results for food' do
+      visit root_path
+
+      fill_in :q, with: 'sweet potatoes'
+      click_button 'Search'
+
+      expect(current_path).to eq(foods_path)
+      
+      expect(page.status_code).to eq 200
+      expect(page).to have_content("Total items returned: 44128")
+      expect(page).to have_content("GTIN/UPC code:")
+      expect(page).to have_content("Description:")
+      expect(page).to have_content("Brand Owner:")
+      expect(page).to have_content("Ingredients:")
+    end
```

Comment:
Solid test, but could be more robust and improved with `within` blocks, counting the number of `Food` elements on the page which should be 10, and thinking about a sad path. Also, we should be stubbing the API call.